### PR TITLE
Fix make validate

### PR DESCRIPTION
**Pull request by kakkoyun**
Signed-off-by: Kemal Akkoyun <kakkoyun@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -113,3 +118,14 @@ clean:
 	find resources/observability/prometheusrules -type f ! -name '*.yaml' -delete
 	find resources/observability/grafana/observatorium -type f ! -name '*.yaml' -delete
 	find resources/observability/grafana/observatorium-logs -type f ! -name '*.yaml' -delete
+
+# Tools
+$(TMP_DIR):
```
Comment (squat) [inline Makefile:123]: Looks like this and BIN_DIR don't appear as dependencies of anything, do this code won't get executed, no? Should we make it an ordering dependency of a target?

**Comment by squat** [comment]
Amazing thank you :))

File: Makefile (hunk 1 above)
Comment (kakkoyun) [reply Makefile:123]: BIN_DIR used in OC target but you're right we're missing TMP_DIR adding it now.

Comment (kakkoyun) [reply Makefile:123]: I guess it just worked because of `mkdir -p`

**Pull request merged**